modules/mini-games/Race.lua

```diff
@@ -185,6 +189,7 @@ end
 --- When a player joins place them into their car
 local function on_player_joined(event)
     local player = game.players[event.player_index]
+    player.teleport({0,0}, surface[1])
```

Comment:
Surface is not defined.

Reply:
Was due to config not being found.

Reply:
This line could be `player.teleport(pos, car.surface)` Im not sure if its noticable in game but this would avoid a position change when the player is set as the driver of the car.